.gitignore

```diff
@@ -21,3 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# IDE
+.idea/
+.vscode/
```

Comment:
No need to push xxx-lock.json into the repo. It will be auto-generated when installing

Reply:
One more remark: Could you please move your login and signup js into components folder?